recipe/meta.yaml

```diff
@@ -1,75 +1,93 @@
 {% set name = "bentoml" %}
-{% set version = "0.13.1" %}
+{% set version = "1.0.0" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/BentoML-{{ version }}.tar.gz
-  sha256: 2f53bd2b659e305a97f5cb46fe97f92f65416959e29199fdc55b7637bf7f247b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bentoml-{{ version }}.tar.gz
+  sha256: 6bc4a4bb8f53b6ee70d1169ee439d89680f96771fbb3f11fb5eba13f96598c88
 
 build:
   number: 0
   noarch: python
   entry_points:
-    - bentoml=bentoml:commandline_interface
+    - bentoml=bentoml._internal.cli:cli
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
     - python >=3.7
+    - setuptools >=59.0
+    - setuptools-scm >=6.2.3
+    - wheel  
   run:
     - python >=3.7
-    - aiohttp
-    - aiohttp-cors ==0.7.0
-    - alembic
-    - boto3
-    - cerberus
-    - certifi
+    - aiofiles
+    - aiohttp <=3.8.1
+    - attrs >=21.1.0
+    - backports.cached-property
+    # NOTE: cattrs has not been updated (last version: 1.10.0)
+    # - cattrs >=21.1.0
+    - cattrs >=1.10.0
     - chardet
+    - circus
     - click >=7.0
-    - configparser
+    - cloudpickle
     - deepmerge
-    - docker-py
-    - flask
-    - grpcio
-    - gunicorn
-    - humanfriendly
+    # NOTE: exceptiongroup is available on channel: conda-forge/label/exceptiongroup_dev
+    - exceptiongroup
+    - filelock
+    - fs
+    - importlib-metadata
+    - jinja2 >=3.0.1
     - numpy
-    - packaging
-    - prometheus_client
-    - protobuf >=3.8.0
+    - opentelemetry-api ==1.9.0
+    - opentelemetry-instrumentation ==0.28b0
+    - opentelemetry-instrumentation-aiohttp-client ==0.28b0
+    - opentelemetry-instrumentation-asgi ==0.28b0
+    - opentelemetry-sdk ==1.9.0
+    - opentelemetry-semantic-conventions ==0.28b0
+    - opentelemetry-util-http ==0.28b0
+    - packaging >=20.0
+    - pathspec
+    - pip-tools >=6.6.2
+    - prometheus_client >=0.10.0,<0.14.0
     - psutil
-    - python-dateutil >=2.7.3,<3.0.0
+    - pynvml <12
+    - python-dateutil
+    - python-dotenv >=0.20.0
     - python-json-logger
+    - python-multipart
+    - pyyaml >=5.0
     - requests
-    - ruamel.yaml >=0.15.0
+    - rich >=11.2.0
     - schema
-    - simple-di
-    - sqlalchemy >=1.3.0,<1.4.0
-    - sqlalchemy-utils <0.36.8
-    - tabulate
-    - urllib3 <=1.25.11
+    - simple-di >=0.1.4
+    - starlette
+    - typing-extensions >=4.0
     ## Converting the recipe to noarch: only build for python>=3.7
```

Comment:
```suggestion
```

Reply:
I would prefer to keep this comment for posterity. If you use `grayskull` utility, you will see it does not suggest using `noarch`. But from my experience of authoring and maintaining close to around 60 packages on conda-forge, I would suggest you to always prefer using `noarch` packages. In this case you only need to successfully build it on linux and it will be available for all other platforms.